webpack.config.dev.js

```diff
@@ -7,23 +7,28 @@ module.exports = {
     "xfc": ["babel-polyfill", "./src"],
   },
   output: {
-    path: "./dist/",
     filename: "xfc.js",
     libraryTarget: 'umd',
     library: 'XFC'
   },
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.js$/,
-        include: /src|index\.js/,
-        loader: 'babel-loader',
-        query: {
-          presets: ["es2015"],
-        }
+        include: path.resolve(__dirname, 'src'),
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: ["es2015"],
```

Comment:
Oh, didn't even know that webpack's babel-loader will use .babelrc file. Updated based on your suggestion. Thanks!